test/wasm_test.rb

```diff
@@ -13,6 +13,30 @@ def test_square
     assert_equal(9, result)
   end
 
+  def test_and
+    instance = instantiate("and.wasm")
+    assert_equal(false, instance.exports.and(false, false))
+    assert_equal(false, instance.exports.and(false, true))
+    assert_equal(false, instance.exports.and(true, false))
+    assert_equal(true, instance.exports.and(true, true))
```

Comment:
The values passed need to be i32 values because "and", "or" and "xor" implement i32 operations.

Reply:
@edvakf 9c8a4a8 fixed!